Java/Java.sublime-syntax

```diff
@@ -220,7 +220,7 @@ contexts:
 
   anonymous-classes-and-new:
     - match: \bnew\b
-      scope: keyword.control.new.java
+      scope: keyword.operator.word.new.java
```

Comment:
`new` seems like a genuine operator to me (especially in dynamic languages like JavaScript).

Reply:
... you mean `keyword.operator.storage`? ... semantically better than `word`, but would assign it the same color as the preceeding `=` with most color schemes. ... not sure.

This is a try to find a scope which better matches the use in other languages. Found this in JavaScript. 

_Basically, I don't really like the `.word.` sub scope as it is currently used, because it is a so much different meaning then `logical`, ... which live in the same level of scopes._

_It should either be removed or paired with it's counterpart `unary`._

Example:

`keyword.operator.unary.logical` vs. `keyword.operator.word.logical` instead of `keyword.operator.logical`.

Reply:
(I also dislike the (recommended) usage for `.word.`, but I definitely see the uses case. It's just that the demands are conflicting and basically require adding two `keyword.operator` scopes for each token due to the lack of wildcard selectors.)

`unary` vs `binary` is an idea I haven't heard before (and also [not in use anywhere, currently](https://github.com/sublimehq/Packages/search?q=unary&unscoped_q=unary)). It's probably worth talking about, but not within this PR.

Regading `new`, I initially did mean `storage.…`, because it's a keyword that relates to storage operation in that it allocates memory in pretty much every language that uses it (even in JavaScript it's relevant), so it has more importance than a conventional (unary) operator. In addition, it could be applied to functions like `malloc` in addition to their `variable.function` scope, but that might be too invasive for some.

`keyword.operator.storage` would be a weaker form of that (and `variable.function.storage`), but I don't think any color scheme is going to target these, so we might as well use `_.word` until we have a better solution in mind.

Reply:
With C++ in mind `new` _(and `delete`)_ are `keywords`. The `storage.` scope should be left for use with data types only, I think. As of its special meaning, I wouldn't call it a normal function like `malloc` as well.

Reply:
`keyword.storage`? :duck:

Reply:
Sounds reasonable. Some syntaxes use `keyword.other[.new]` already. So `keyword.other.storage.new` would be fine. Not sure if there are enough use cases to make `storage` a 2nd level scope, though. Would wait for an agreement of @wbond in that case.